core/src/main/java/com/linecorp/armeria/server/docs/DocServicePlugin.java

```diff
@@ -24,12 +24,14 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 
+import com.linecorp.armeria.common.util.UnstableApi;
 import com.linecorp.armeria.server.Service;
 import com.linecorp.armeria.server.ServiceConfig;
 
 /**
  * Generates the {@link ServiceSpecification}s of the supported {@link Service}s.
  */
+@UnstableApi
```

Comment:
Should we add to `docs/package-inf.java`?

Reply:
No, because we have `DocService` there. :wink: